pwiz_tools/Skyline/Model/Databinding/Entities/TransitionResult.cs

```diff
@@ -116,6 +116,22 @@ public string Note
             }
         }
 
+        public bool TransitionResultIsQuantitative
+        {
+            get
+            {
+                return Transition.DocNode.IsQuantitative(SrmDocument.Settings);
+            }
+        }
+
+        public bool TransitionResultIsMs1
+        {
+            get
+            {
+                return SrmDocument.Settings.GetChromSource(Transition.DocNode) == ChromSource.ms1;

```

Comment:
Probably this should be != ms2, especially since you widened SIM to be <1000 m/z. It is very possible that MS1 spectra show up with ChromSource.sim.